code/matt_r/python/lab_2b_palindrome.py

```diff
@@ -0,0 +1,46 @@
+
+# palindrome
+
+
+# # word = 'racecar'
+# # word = "newb"
+
+
+def check_palindrome(word):
+    word_list = list(word)
+    word_list_2 = list(word)
+    word_list_2.reverse()
+    word_list == word_list_2
+
+    if word_list == word_list_2:
+       return print(f'{word} is a palindrome!')
+
+    else:
+        return print(f'{word} is not a palindrome!')
+
+
+word = input('Enter a word to see if its a palindrome:  ') 
+
+
+check_palindrome(word)
+print(f'Thanks for playing,')
+
+
+#   anagram
+def anagram(first, second):
+    first_2 = list(first)
+    second_2 = list(second)
+    first_2.sort()
+    second_2.sort()
+
+    if first_2 == second_2:
+        return print(f'{first} and {second} are anagrams')
+    else:
+            return print(f'{first} and {second} are not anagrams')
+
+
+        
+first = input('Enter the first word:  ')
+second = input('Enter the second word:  ')
+
+anagram(first, second)
```

Comment:
try testing with the phrases:

`anagram` and `nag a ram`  (replace function)

`anagram` and `Anagram`  (lower method)

both should be anagrams.

Reply:
Check assignment for missing methods.